.devcontainer/devcontainer.json

```diff
@@ -17,7 +17,7 @@
     "naumovs.color-highlight",
     "prettier.prettier-vscode",
     "stkb.rewrap",
-    "stylelint.vscode-stylelint",
+    // "stylelint.vscode-stylelint",
```

Comment:
Not valid JSON.

Reply:
True... but I'm not sure why that matters? There was already a comment at the top of the file; `devcontainer.json` is not required to be strict JSON.

Reply:
That's part of why I marked this "approved" anyway; it was more a "btw, in case this matters to you" comment 🤷‍♀